make/fix_imports.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I think you want to put this script in `hack/` directory. We already do this kind of thing in our makefiles - specifically, see:

https://github.com/kiali/kiali/blob/master/make/Makefile.build.mk#L20

where we call `hack/check_go_version.sh` to make sure the make build has a supported Go version. That script is here in the hack directory: https://github.com/kiali/kiali/blob/master/hack/check_go_version.sh

Other than that, all the imports look prestine now :)

Reply:
done!